File: easy/last_word_len/last_word_len.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        s = s.rstrip()
        i = len(s)-1
        length_of_word = 0
        while i >= 0 and s[i] != ' ':
            length_of_word += 1
            i -= 1
        return length_of_word


s = Solution()
print("Solution 1 : ", s.lengthOfLastWord("Hello World"))

File: easy/last_word_len/last_word_len.go
package main

import (
	"fmt"
	"strings"
)

func lengthOfLastWord(s string) int {
	lengthOfWord := 0
	s = strings.TrimSpace(s)

	i := len(s) - 1
	for i >= 0 && s[i] != ' ' {
		lengthOfWord++
		i--
	}

	return lengthOfWord
}

func main() {
	fmt.Println("Solution 1 : ", lengthOfLastWord("Hello World"))
}
